doc/bitcoin-conf.md

```diff
@@ -50,3 +50,11 @@ rpcport=4000
 The configuration file is not automatically created; you can create it using your favorite text editor. By default, the configuration file name is `bitcoin.conf` and it is located in the Bitcoin data directory, but both the Bitcoin data directory and the configuration file path may be changed using the `-datadir` and `-conf` command-line options.
 
 The `includeconf=<file>` option in the `bitcoin.conf` file can be used to include additional configuration files.
+
+### Default configuration file locations
+
+Operating System | Data Directory | Example Path
+-- | -- | --
+Windows | %APPDATA%\Bitcoin\ | C:\Users\username\AppData\Roaming\Bitcoin\bitcoin.conf
```

Comment:
Please escape these paths using backtips e.g. `C:\Users\username\AppData\Roaming\Bitcoin\bitcoin.conf`. I think this rendering is more appropriate, and also, otherwise, `'\x` will be seen as markdown escaping.

Reply:
thank you, added backticks